core/src/test/resources/org/elasticsearch/index/query/not-filter3.json

```diff
@@ -1,7 +0,0 @@
-{
-            "not":{
-                "term":{
-                    "name.first":"shay1"
```

Comment:
Ah, think I got it, this is all handled by the alternateVersions in the NotQueryBuilderTests? Forgot that we introduced them there.